### DISCUSSION TITLE
Auto Fleet rotation after level run is complete.

### DISCUSSION THREAD

**Discussion by mikerod123** [LmeSzinc/AzurLaneAutoScript#3690] [Ideas]
Auto switch fleets to help preserve happiness for longer farming level runs, have fleets switched out when 1 or more ship girls become unhappy or switch out after every run. mob fleets get switched out more often than boss fleets.

Setup mob fleet rotation selection FL1>FL2>FL3>FL4 in that rotation or mix it up.

Setup boss rotation FL5>FL6 if you want to not use a certain fleet just leave it off.

I can see a default option like (switch after every run) or (switch after ship girls become unhappy) and the other one being togglable.
And maybe an option for if you want to use only Mob fleet for the whole run FL1>FL2>FL3>FL4>FL5>FL6
But only works with normal maps/levels where you don't make the temporary fleet like in.

**Comment by LmeSzinc** [comment]
There are 3 main tasks and 2 event tasks enough for fleet combinations. 3 is the maximum number of mood scheduling, the first task's mood is restored while waiting for the third, so the fourth task never runs.
